packages/next/src/client/components/react-dev-overlay/internal/helpers/stitched-error.ts

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+const captureOwnerStack = process.env.__NEXT_REACT_OWNER_STACK
+  ? (React as any).captureOwnerStack
+  : () => ''
+
+const REACT_ERROR_STACK_BOTTOM_FRAME = 'react-stack-bottom-frame'
+const REACT_ERROR_STACK_BOTTOM_FRAME_REGEX = new RegExp(
+  `(at ${REACT_ERROR_STACK_BOTTOM_FRAME} )|(${REACT_ERROR_STACK_BOTTOM_FRAME}\\@)`
+)
+
+export function getReactStitchedError<T = unknown>(err: T): Error {
+  if (!process.env.__NEXT_REACT_OWNER_STACK) {
+    return err as any
+  }
+
+  const isErrorInstance = err instanceof Error
```

Comment:
This is likely inaccurate. Especially client-side due to cross-realm errors. We use `isError` throughout the codebase which is looser than it needs to be now that React Flight deserializes Error properly but it's at least consistently too loose.